src/translations/en.json

```diff
@@ -7357,6 +7357,8 @@
           },
           "elements": {
             "style": "Style",
+            "left": "[%key:ui::panel::page-demo::config::arsaboo::names::left%]",
+            "top": "[%key:ui::panel::lovelace::editor::edit_view_header::settings::badges_position_options::top%]",
```

Comment:
The UX team will have a look at the labels but we shouldn't use completely unrelated ones like these even they have to desired words

Reply:
Okay, I'll create new ones called "Horizontal Position" and "Vertical Position".